docs/zh/UI/quickStart/transform.md

```
@@ -0,0 +1,34 @@
+title: UITransform
+type: UI
+label: UI
+---
+
+`UITransform` 组件是专门设计用来表示 UI 元素的尺寸和位置的，它继承于 [Transform](/apis/core/#Transform) 。
+
+## 编辑器使用
+
+添加了 UI 组件的节点，会自动添加 `UITransform` 组件（替换原先旧的 [Transform](/apis/core/#Transform) 组件），在编辑器中，可以选中节点可以使用 `RectTool` （快捷键 `T` ）快速设置属性，也可以在在 **[检查器面板](/docs/interface/inspector)** 设置精确属性。
+
+<img src="https://mdn.alipayobjects.com/huamei_yo47yq/afts/img/A*hJ81TKtDKLIAAAAAAAAAAAAAehuCAQ/original" style="zoom:50%;" />
```

Markdownlint (0.37.0)

Images should have alternate text (alt text)

(MD045, no-alt-text)